crates/sqlbuiltins/src/builtins.rs

```diff
@@ -689,6 +689,20 @@ FROM (VALUES (NULL, NULL, NULL, NULL,
     NULL, NULL, NULL, NULL, NULL)) WHERE false",
 });
 
+pub static PG_MATVIEWS: Lazy<BuiltinView> = Lazy::new(|| BuiltinView {
+    schema: POSTGRES_SCHEMA,
+    name: "pg_matviews",
+    sql: "
+    SELECT '' as schemaname,
+    '' as matviewname,
+    '' as matviewowner,
+    '' as tablespace,
+    false as hasindexes,
+    false as ispopulated,
+    '' as definition
```

Comment:
```suggestion
    SELECT '' as schemaname,
    '' as matviewname,
    '' as matviewowner,
    '' as tablespace,
    false as hasindexes,
    false as ispopulated,
    '' as definition
    WHERE false
```

To avoid returning a row.